rats/pyproject.toml

```diff
@@ -20,13 +20,9 @@ dirty = false
 # https://developercommunity.visualstudio.com/t/local-version-segments-for-python-package-feeds/892057
 metadata = false
 
-[tool.poetry-monorepo-dependency-plugin]
-enable = true
-version-pinning-strategy = "semver"
-
 [tool.poetry.dependencies]
-rats-pipelines = { path = "../rats-pipelines", develop = true }
-rats-processors = { path = "../rats-processors", develop = true }
+rats-pipelines = "*"
+rats-processors = "*"
```

Comment:
I am not too happy with these, so let's figure out what the issue is, with time... Definitely not a priority.

Reply:
for the future, the main things that broke with the monorepo plugin were related to using non-poetry commands on our components. `pip` and `pipx` installs should work without needing to call poetry. this was especially causing issues when trying to test a `dev#` version of a package because the monorepo plugin created a dependency that was impossible to meet with a package that did not yet exist (devtools installing apps).

Reply:
the monorepo plugin is only called when building wheels, AFAIK. Once they are built and published, you can install them however you want, even if they are dev versions. So not sure what the problem is exactly.